Documentation/Relationship/Relationship.md

```diff
@@ -0,0 +1,334 @@
+# Retrieving Relationship Information
+
+This proposal describes how relationship information can be generated from the DSL, introduces some concepts
+to give hints for relationship retrieval and adds mechanisms for manually defined hints.
+
+A potential REST interface exporter will use such relationship information to generate Hypermedia information.
+Specifically it will generate hyperlinks which SHOULD be place in a designated `_links` section.
```

Comment:
```suggestion
Specifically it will generate hyperlinks which SHOULD be placed in a designated `_links` section.
```

Reply:
Thanks, will correct that in the next commit I'll do